Utils/adopt_utils.bash

```diff
@@ -315,11 +319,24 @@ add_msg_to_readme(){
 	message=$2
 	os=$(detect_os)
 
-	if [ "$os" == "Mac OS" ] 
+	# Check if README exists, create if not
+	[ ! -f "$readme" ] && touch "$FILE"
+
+	if [ "$os" == "Mac OS" ]
 	then
-		sed -i '' "1s/^/$message\n\n/" "$readme"
+		if ! [[ -s "$readme" ]]; then
+        	# The file is empty.
+			echo "$message" > "$readme"
```

Comment:
We can have this code part running for al cases and only if`[ "$os" == "Mac OS" ]` then do: `sed -i '' "1s/^/$message\n\n/" "$readme"` else do: `sed -i "1s/^/$message\n\n/" "$readme"`.
i.e we only have 1 line which is different - better practice

Reply:
Makes sense, logic changed in [`04b254d` (#21686)](https://github.com/demisto/content/pull/21686/commits/04b254d32efc55887046036d12ee2924c1c2ccd0).

Tested function after changes:
```bash
source ./Utils/adopt_utils.bash

# Case 1: README doesn't exist
> cat /tmp/README.md
cat: /tmp/README.md: No such file or directory

> add_msg_to_readme  /tmp/README.md "- Adoption started."

> cat /tmp/README.md
- Adoption started.

# Case 2: README exists but is empty

> /tmp/README.md

> add_msg_to_readme  /tmp/README.md "- Adoption started."

> cat /tmp/README.md
- Adoption started.

# Case 3: README exists and is not empty

> add_msg_to_readme ./Packs/HelloWorld/README.md  "- Adoption started."

> tail ./Packs/HelloWorld/README.md head -2

This content pack provides the Feed Hello World integration for getting started with your feed integration and the Hello World integration that imports events as incidents.

- Adoption started.

```